esp32/PermanentMarker36.c

```diff
@@ -0,0 +1 @@
+../../components/ugfx/PermanentMarker36.c
```

Comment:
No please don't move them, then it won't work with pure ugfx (without python) anymore :).

it'd be nice if upy could use the ugfx fonts without these symlinks, but that might be hard for some reason. Anyway this is OK for now.